core/Plugin/Manager.php

```diff
@@ -44,6 +44,8 @@ class Manager extends Singleton
 
     protected $doLoadPlugins = true;
 
+    private $pluginsLoadedAndActivated;
```

Comment:
At some point we can load this into Piwik\Cache\Transient but even transient cache is slow when called millions of times. Same applies to some other static caches that I used. We'll need to improve performance of transient cache at some point.